core/templates/base.html

```diff
@@ -1,13 +1,41 @@
+{% load static %}
+
 <!DOCTYPE html>
 <html lang="en">
     {% include "html_head_tag.html" %}
-    <body>
+    <body id="document-body">
         {% include "header.html" %}
+
         <main>
             <div class="container">
                 {% block content %}{% endblock %}
             </div>
         </main>
+
         {% include "footer.html" %}
+
+        <!-- Back to top button -->
+        <!-- href is added as a fallback - works without JS -->
```

Comment:
We use only `href` (i.e. no JS) to scroll back, so this comment is not relevant anymore ? ^^